### PR TITLE
fixed last version

### PR DESCRIPTION
**Pull request by LebedevOleksii**

### CODE REVIEW

File: user.js [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
-    render(res)
-})
-.catch((err)=>console.log(err));
+(function request(){
+    let params = new URL(window.location.href).searchParams;
+    const id = params.get('id');
+    
+    fetch(`https://test-users-api.herokuapp.com/users/${id}`)
+    .then(response => response.json())
+    .then(response=> res = response.data)
```
Comment (mmaksymovych) [inline user.js]: change to `response => render(response.data)`

File: user.js [hunk 2]
```diff
@@ -1,24 +1,15 @@
...
+    .then(res=>render(res))
```
Comment (mmaksymovych) [inline user.js]: remove `.then(res=>render(res))`, you don't need it

File: user.js [hunk 3]
```diff
@@ -1,24 +1,15 @@
-let params = new URL(window.location.href).searchParams;
-const id = params.get('id');
-
-const buttonMenu = document.querySelector('#button-menu')
-buttonMenu.addEventListener('click', function(){
-    window.location.href=`index.html`;
-})
 let res;
```
Comment (mmaksymovych) [inline user.js]: remove this, no need

File: user.html [hunk 4]
```diff
@@ -10,7 +10,7 @@
     <div class="wrapper">
         <span><h3>Інформація про користувача</h3></span>
         <div id="user-info"></div>
-        <button id="button-menu">Меню</button>
+        <a href="index.html">На головну</a>
```
Comment (mmaksymovych) [inline user.html]: add class and attach styles to class, not to tag

File: style.css [hunk 5]
```diff
@@ -17,47 +17,47 @@
...
 }
-#go-up{
+#go-up {
     position: absolute;
     text-decoration-line: none;
     color: rgb(153, 147, 147);
     margin: 110px 132px;
 }
-a{
+a {
```
Comment (mmaksymovych) [inline style.css]: use class please

File: ajaxHW2.js [hunk 6]
```diff
@@ -181,9 +181,9 @@ const addIcon = new Icon({
     width: 50,                                 
     heigt: 50,    
     parentElement: form,                           
-    onClick: function(event){ 
+    onClick: function(event) {
```
Comment (mmaksymovych) [inline ajaxHW2.js]: remove **event**, you are not using it

File: ajaxHW2.js [hunk 7]
```diff
@@ -114,55 +114,55 @@ function renderUsers(){  // відображення юзерів
...
         usersList[index] = response.data;
         renderUsers()
-    } catch(err){
+    } catch(err) {
         console.log(err)
     }
 }
-const deleteUser = async (event, id)=>{   // DELETE
-    try{
+const deleteUser = async (id)=> {   // DELETE
```
Comment (mmaksymovych) [inline ajaxHW2.js]: no need for brackets '()' change to `= async id => {}`, and add space before =>

File: ajaxHW2.js [hunk 8]
```diff
@@ -114,55 +114,55 @@ function renderUsers(){  // відображення юзерів
...
         const response = await request('users/');
         usersList = response.data;
         renderUsers()
-    } catch(err){
+    } catch(err) {
         console.log(err)
     }
 }
-const editUser = async (event, id, index)=>{ // PUT
+const editUser = async (id, index)=> { // PUT
```
Comment (mmaksymovych) [inline ajaxHW2.js]: add space before =>

File: ajaxHW2.js [hunk 9]
```diff
@@ -114,55 +114,55 @@ function renderUsers(){  // відображення юзерів
...
         const response = await request('users/');
         usersList = response.data;
         renderUsers()
-    } catch(err){
+    } catch(err) {
         console.log(err)
     }
 }
-const addUser = async ()=>{  // POST
+const addUser = async ()=> {  // POST
```
Comment (mmaksymovych) [inline ajaxHW2.js]: add space before =>

File: ajaxHW2.js [hunk 10]
```diff
@@ -114,55 +114,55 @@ function renderUsers(){  // відображення юзерів
...
     
-    usersList.forEach((user, index)=>{
+    usersList.forEach((user, index)=> {
     renderOneUser(user, index);
     });
 };
 
-const getUsers = async ()=>{ // GET
-    try{
+const getUsers = async ()=> { // GET
```
Comment (mmaksymovych) [inline ajaxHW2.js]: add space before =>

File: ajaxHW2.js [hunk 11]
```diff
@@ -104,7 +104,7 @@ function renderUsers(){  // відображення юзерів
             width: 34,                                 
             heigt: 34,    
             parentElement: userButtons,                           
-            onClick: function(event){          
+            onClick: function() {
```
Comment (mmaksymovych) [inline ajaxHW2.js:107]: user arrow functions `() => {}`

File: ajaxHW2.js [hunk 12]
```diff
@@ -95,7 +95,7 @@ function renderUsers(){  // відображення юзерів
             width: 34,                                 
             heigt: 34,    
             parentElement: userButtons,                           
-            onClick: function(event){              
+            onClick: function() {
```
Comment (mmaksymovych) [inline ajaxHW2.js:98]: use arrow functions `() => {}`

File: ajaxHW2.js [hunk 13]
```diff
@@ -86,7 +86,7 @@ function renderUsers(){  // відображення юзерів
...
```
Comment (mmaksymovych) [inline ajaxHW2.js:89]: use arrow functions () => {}

File: ajaxHW2.js [hunk 14]
```diff
@@ -19,26 +19,26 @@ class Icon {            // іконки кнопки
...
-    appendToElement(element){       
+    appendToElement(element) {       
         element.append(this.icon);
     }
 
-    animateIcon(){         
+    animateIcon() {         
         this.icon.style.opacity = 0;  
-        setTimeout(()=>{
+        setTimeout(()=> {
```
Comment (mmaksymovych) [inline ajaxHW2.js]: add space before =>

File: ajaxHW2.js [hunk 15]
```diff
@@ -19,26 +19,26 @@ class Icon {            // іконки кнопки
         this.icon.style.transition = `opacity .5s`; 
         this.icon.style.cursor = 'pointer';   
         this.icon.src = source;                      
-        this.icon.addEventListener('click',(event)=>{        
+        this.icon.addEventListener('click',(event)=> {
```
Comment (mmaksymovych) [inline ajaxHW2.js]: add space before =>

File: ajaxHW2.js [hunk 16]
```diff
@@ -19,26 +19,26 @@ class Icon {            // іконки кнопки
...
-        setTimeout(()=>{
+        setTimeout(()=> {
             this.icon.style.opacity = 1;
         },300);
     }
 }
 
 
-const request = function (endpoint, method, data){    //  запит
+const request = function (endpoint, method, data) {    //  запит
```
Comment (mmaksymovych) [inline ajaxHW2.js]: use arrow function  `const request = (endpoint, method, data) => {}`

File: ajaxHW2.js [hunk 17]
```diff
@@ -55,7 +55,7 @@ const request = function (endpoint, method, data){    //  запит
 let usersList = [];
 
 function renderUsers(){  // відображення юзерів
```
Comment (mmaksymovych) [inline ajaxHW2.js]: use arrow functions `const renderUsers = () => {}`

File: ajaxHW2.js [hunk 18]
```diff
@@ -55,7 +55,7 @@ const request = function (endpoint, method, data){    //  запит
...
-    function renderOneUser(user, index){
+    function renderOneUser(user, index) {
```
Comment (mmaksymovych) [inline ajaxHW2.js:58]: use arrow functions `const renderOneUser = () => {}`

File: ajaxHW2.js [hunk 19]
```diff
@@ -114,55 +114,55 @@ function renderUsers(){  // відображення юзерів
     const wrapperDiv = document.querySelector('#wrapper');    
     wrapperDiv.innerHTML = "";
     
-    usersList.forEach((user, index)=>{
+    usersList.forEach((user, index)=> {
```
Comment (mmaksymovych) [inline ajaxHW2.js]: add space before =>

File: ajaxHW2.js (hunk 6 above)
Comment (mmaksymovych) [reply ajaxHW2.js]: and use arrow function

File: ajaxHW2.js [hunk 20]
```diff
@@ -181,9 +181,9 @@ const addIcon = new Icon({
     width: 50,                                 
     heigt: 50,    
     parentElement: form,                           
-    onClick: function(event){ 
+    onClick: function(event) { 
         addUser()
-        setTimeout(()=>{
+        setTimeout(()=> {
```
Comment (mmaksymovych) [inline ajaxHW2.js]: space before =>

File: user.js [hunk 21]
```diff
@@ -1,24 +1,15 @@
...
-.catch((err)=>console.log(err));
+(function request(){
+    let params = new URL(window.location.href).searchParams;
+    const id = params.get('id');
+    
+    fetch(`https://test-users-api.herokuapp.com/users/${id}`)
+    .then(response => response.json())
+    .then(response=> res = response.data)
+    .then(res=>render(res))
+    .catch(err=>console.log(err));
```
Comment (mmaksymovych) [inline user.js]: spaces

File: user.js [hunk 22]
```diff
@@ -1,24 +1,15 @@
...
+})();
 
 function render(){
```
Comment (mmaksymovych) [inline user.js]: use arrow function `const render = () => {}`

**Review by mmaksymovych — CHANGES_REQUESTED** [review]
Looks good, just some styling comments. 
Don't forget about spaces and general code style, read this - https://learn.javascript.ru/coding-style some good tips :)
Also, try to use es6 syntax as much as possible, like arrow function instead of regular etc.
You can read more here https://habr.com/post/305900/ - good article

File: user.js (hunk 1 above)
Comment (LebedevOleksii) [reply user.js]: тоді в мене не виходить 
<span>Імя: ${response.data.name}</span> <br/>
        <span>Вік: ${response.data.age}</span> <br/>
        <span>ID: ${response.data.id}</span> <br/>

**Pull request merged**